pom.xml

```diff
@@ -8,63 +8,62 @@
   <packaging>jar</packaging>
   <version>1.1.1</version>
   <name>ItemExchange</name>
-  <url>https://github.com/Civcraft/ItemExchange</url>
+  <url>https://github.com/DevotedMC/ItemExchange</url>
   
   <properties>
-        <maven.compiler.source>1.7</maven.compiler.source>
-        <maven.compiler.target>1.7</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
   </properties>
   
-  <build>
-    <sourceDirectory>${basedir}/src</sourceDirectory>
-    
+  <build>    
     <resources>
       <resource>
         <directory>${basedir}</directory>
         <includes>
           <include>license.txt</include>
+          <!--If a plugin has a decent README for once, let's include it -->
+          <include>README.md</include>
         </includes>
       </resource>
       <resource>
-        <directory>${basedir}/src</directory>
+        <directory>${basedir}/src/main/resources</directory>
         <includes>
-          <include>**/*.java</include>
```

Comment:
I think someone was trying to include the sources in with the distribution jar, check the license, might be a requirement.